src/mesh/mesh.cpp

```diff
@@ -1086,24 +1086,53 @@ void Mesh::Initialize(bool init_problem, ParameterInput *pin, ApplicationInput *
     }
 
     // send FillGhost variables
+    bool can_delete;
+    do {
```

Comment:
Should these do-while loops have some upper iteration count in case something goes really wrong?

Reply:
Yeah, probably